CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+Issue #548 : added functionality: persist attribute persistence in column using mongo-sink
```

Comment:
This must be:

    - [FEATURE] Column-like persistence in OrionMongoSink (#548)

Thanks! :)